common/Makefile.common.mk

```diff
@@ -116,6 +116,6 @@ tidy-docker:
 
 # help works by looking over all Makefile includes matching `target: ## comment` regex and outputting them
 help: ## Show this help
-	@egrep -h '^[a-zA-Z_\.-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort  | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
```

Comment:
we can move this target to `Makefile.overrides.mk`, we just need to manually define MAKEFILE_LIST as well then:

```

MAKEFILE_LIST = Makefile Makefile.core.mk Makefile.overrides.mk

.PHONY: help
help: ## Print this help
	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf " \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
```

That way it works and we don't need to change `Makefile.common.mk` or `Makefile`